randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -152,7 +152,12 @@ Extra - asset_id: 3
           Any of the following:
               # Via bottom path
               Can Lay Bombs
-              Morph Ball and Enabled Moat Tweaks
+              All of the following:
+                  Morph Ball
+                  Any of the following:
+                      Enabled Moat Tweaks
+                      # Destroy with Screw
+                      Gravity Suit and Screw Attack
 
```

Comment:
I don't know if this can be done with screw. I've tried to break the block in the SM Practice Hack and it doesn't work.

Reply:
Then i'll go remove them